packages/argo-checkout/package.json

```diff
@@ -2,6 +2,11 @@
   "name": "@shopify/argo-checkout",
   "description": "The API for Argo extensions that run in Shopify’s Checkout",
   "version": "0.7.0",
+  "repository": {
```

Comment:
Interesting. Should we be doing this for all packages in this repo?

Reply:
I haven't had a chance to test this change since I think it requires publishing a scoped package. Based on [reading the npm docs](https://docs.npmjs.com/files/package.json.html#repository), it seems like a reasonable configuration to include.

I can add it to the other packages if you would like.